src/api.raml

```diff
@@ -191,6 +191,51 @@ traits:
               required: false
               example: [article/1234, interview/5678]
 
+/annotations:
+    (experimental):
+    type: collection
+    description: |
+        Annotations
+    get:
+        description: |
+            Get a list of annotations, sorted by date.
+        is:
+          - paged: { resources: Annotations }
+        queryParameters:
+            by:
+                description: |
+                    Only return annotations by this author.
+                type: string[]
+                required: false
```

Comment:
I've set this to be optional now.

Reply:
I didn't think we were going to list _all_ annotations? This has the same author problem (you don't know whose annotation it is).

Reply:
Right, I'll set it back to required.